tmt/steps/report/reportportal.py

```diff
@@ -102,12 +108,10 @@ class ReportReportPortal(tmt.steps.report.ReportPlugin[ReportReportPortalData]):
     For communication with Report Portal API is neccessary to provide
     following options:
 
-    * token for authentication
+    * TOKEN for authentication
     * URL of the ReportPortal instance
-    * project name
-    * optional API version to override the default one (v1)
-    * optional launch name to override the deafult name based on the tmt
-      plan name
+    * PROJECT name
+    * [LAUNCH] name to override the default name based on the tmt plan name
```

Comment:
Please, don't undo these changes. I tried to improve the docstring, aligning the plugin more with the rest of the flock.

For example, there is no such thing as "PROJECT" - there is `--project` option, there is a `project` key, and both refer to the `PROJECT_NAME` metavar. There's no "PROJECT", in capital letters. On the other hand, one should provide a "project name" - either by option or a key, hence it's not referring to one particular way on purpose - token, and optionally "launch name" or "API version".

`[LAUNCH]` is again very hard to follow: there are `--launch` and `launch` and `LAUNCH_NAME`, but no `LAUNCH` in the help texts of plugin keys/options. Plus its optionality is already expressed in the text - `[]` is commonly used with CLI options, but may not be that clear for someone editing an fmf file.